feature/home/src/main/kotlin/com/nexters/bandalart/android/feature/home/ui/BottomSheetContent.kt

```diff
@@ -0,0 +1,355 @@
+            text = "마감일을 선택해주세요.",
+            color = Gray400,
+            fontSize = 16.sp,
+            fontFamily = pretendard,
+            fontWeight = FontWeight.W600,
+          )
+          Icon(
+            modifier = Modifier
+              .align(Alignment.CenterEnd)
+              .height(21.dp)
+              .aspectRatio(1f),
+            imageVector = Icons.Default.ArrowForwardIos,
+            contentDescription = "Localized description",
+            tint = Gray400,
+          )
+
+          if (openDatePickerBottomSheet) {
+            ModalBottomSheet(
+              modifier = Modifier.wrapContentSize(),
+              onDismissRequest = { openDatePickerBottomSheet = !openDatePickerBottomSheet },
+              sheetState = datePickerState,
+              windowInsets = WindowInsets.ime,
+              dragHandle = null,
+            ) {
+              DatePickerUI(
+                label = "Date Picker",
+                onDismissRequest = { openDatePickerBottomSheet = !openDatePickerBottomSheet },
+                onResult = { openDatePickerBottomSheet = it },
+                datePickerScope = datePickerScope,
+                datePickerState = datePickerState,
+              )
+            }
+          }
+        }
+      }
+      Spacer(modifier = Modifier
+        .fillMaxWidth()
+        .padding(top = 10.dp)
+        .height(1.dp)
+        .background(color = Gray300)
+      )
+      Column(modifier = Modifier.padding(top = 28.dp)) {
+        var goalName by remember { mutableStateOf("") }
+        Text(
+          text = "메모 (선택)",
+          textAlign = TextAlign.Start,
+          color = Gray600,
+          fontSize = 12.sp,
+          fontFamily = pretendard,
+          fontWeight = FontWeight.W700,
+        )
+        BasicTextField(
+          modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = 12.dp),
+          value = goalName,
+          onValueChange = { goalName = if (it.length > 15) goalName else it },
+          keyboardOptions = KeyboardOptions.Default.copy(imeAction = ImeAction.Done),
+          maxLines = 1,
+          decorationBox = { innerTextField ->
+            Box(modifier = Modifier.fillMaxWidth()) {
+              if (goalName.isEmpty()) {
+                Text(
+                  text = "메모를 입력해주세요.",
+                  color = Gray400,
+                  fontSize = 16.sp,
+                  fontFamily = pretendard,
+                  fontWeight = FontWeight.W600,
+                )
+              }
+              innerTextField()
+            }
+          },
+        )
+      }
+      Spacer(modifier = Modifier
+        .fillMaxWidth()
+        .padding(top = 10.dp)
+        .height(1.dp)
+        .background(color = Gray300)
+      )
+      if (!isSubCell) {
+        Column(modifier = Modifier.padding(top = 28.dp)) {
+          var switchOn by remember { mutableStateOf(false) }
+          Text(
+            text = "달성여부",
```

Comment:
```suggestion
text = "달성 여부",
```
피그마 보면 이거 띄어쓰기 되어있던데 이게 맞지 않나?

Reply:
아 최종 전 피그마에서는 붙어있었네 ㅋㅋ..

Reply:
아 근데 이후에는 띄어쓰기를 했다고 나오는데 혹시 최근 커밋부터 보고 있는거야?

Reply:
반영한거는 아직 안봐서 한번 훑을때 대충 훑어서, 위에서 부터 커밋한거 두번씩 보고있었음